gosam.spec

```diff
@@ -0,0 +1,32 @@
+
+### RPM external gosam 2.0.4-12f4de9
+Source: http://www.hepforge.org/archive/gosam/gosam-%{realversion}.tar.gz
+
+Requires: qgraf
+Requires: form
+Requires: gosamcontrib
+Requires: python cython
+
+
+
+%prep
+%setup -q -n gosam
+
+
+
+%build
+CXX="$(which %{cms_cxx}) -fPIC"
```

Comment:
cms_cxx is not defined. I would suggest to replace it directly with g++. same for gosamcontrib.spec